crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -415,13 +415,18 @@ impl SlidingSync {
     ) -> Result<(http::Request, RequestConfig, OwnedMutexGuard<SlidingSyncPositionMarkers>)> {
         // Collect requests for lists.
         let mut requests_lists = BTreeMap::new();
+        let all_lists_are_fully_loaded;
```

Comment:
Having auxiliary verbs in variable names always feels super weird. How about `fully_loaded_all_lists`?